src/components/Button/ButtonTypes.ts

```diff
@@ -47,6 +47,7 @@ export type OnlyButtonProps = {
   type?: HTMLButtonElement['type'];
   /**
    * ボタンを無効化するかどうか
+   * @default false
```

Comment:
Explicitly state false even for boolean. No need to judge when writing comments.